docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -174,7 +174,7 @@ function su_humsci_profile_simplify_condition_forms(array &$condition_elements,
 /**
  * Implements hook_ENTITY_TYPE_insert().
  */
-function hs_field_helpers_eck_entity_type_insert(EntityInterface $entity) {
+function su_humsci_profile_eck_entity_type_insert(EntityInterface $entity) {
```

Comment:
whoops. How'd we let that one through?

Reply:
and it still worked which was surprising. :man_shrugging: